tools/jenkins-run.sh

```diff
@@ -52,4 +54,4 @@ else
     DOCKER_OPT+=("-v" "${__home}:/var/jenkins_home")
 fi
 
-docker run -d --name ${__name} -p 8080:8080 -p 50000:50000 ${DOCKER_OPT[@]} jenkins
+docker run -d --group-add $(getent group docker | awk -F: '{print $3}') --name ${__name} -p 8080:8080 -p 50000:50000 ${DOCKER_OPT[@]} jenkins
```

Comment:
Why do we need group-add?

Reply:
We need  to add docker guid to user to allow him to use `/var/run/docker.sock`. The problem is that docker guid inside container can be not equal to docker guid on host so we add host's docker guid via ` --group-add`.